molecule/default/molecule.yml

```diff
@@ -1,13 +1,15 @@
 ---
 dependency:
   name: galaxy
-  requirements_file: molecule/default/requirements.yml
+  options:
+    role-file: ansible-role-requirements.yml
 driver:
   name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: molecule/default/yamllint.yml
+lint: |
```

Comment:
If you don't `set -e` at the top here, then the exit code will be determined by the last command run, i.e. it may be 0 although one of the linters failed.

One could also move yamllint before ansible-lint, since yaml errors might throw ansible-lint (not sure)

Reply:
Adding, but will leave the linters in order since it doesn't _really_ matter.